scripts/generate

```diff
@@ -191,7 +191,9 @@ def('substitutions',
                           K(el('span', 'arrow', htmlEncode('~>')))],
                          [contains(_, ap([I, htmlEncode], ['->'])),
                           K(el('span', 'arrow',
-                               el('span', 'arrow-hyphen', htmlEncode('-')) +
+                               //  Use NON-BREAKING HYPHEN so arrow does
+                               //  not break because of line wrapping
+                               el('span', 'arrow-hyphen', '&#x2011;') +
```

Comment:
> what if we just use `→`?

For exactly the reason Serg gave: the character often looks too small. Keep in mind that Transcribe generates *readmes* rather than HTML documents. We have no control over the way GitHub, say, styles a readme. Thus even if we find a typeface with a really nice RIGHTWARDS ARROW we should still have Transcribe use `->`.

I'm open to the possibility of replacing `->` with RIGHTWARDS ARROW in sanctuary-site, as we have full control over styling in that context. I did experiment with that possibility when I first created the site, but I wasn't happy with the results. Note that the `⇒` is wider than the `→` in the screenshot above. We could use a custom font, but that adds complexity to a site which is currently extremely easy to maintain.

Secondarily, I do like getting `->` arrows when copying from the HTML document.